app/models/application_record.rb

```diff
@@ -1,3 +1,6 @@
 class ApplicationRecord < ActiveRecord::Base
   primary_abstract_class
+  def current_user
+    User.first
+  end
```

Comment:
- [ ] This method is supposed to be in the `ApplicationController` please address that

Reply:
This issue is well resolved! ✔️